.github/workflows/ci.yaml

```diff
@@ -141,3 +141,42 @@ jobs:
         run: vendor/bin/phpunit --version
       - name: Run PHPUnit tests
         run: vendor/bin/phpunit
+
+  phpunit-fosuserbundle:
+    name: PHPUnit (PHP ${{ matrix.php }} FOSUserBundle)
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        php:
+          - '7.4'
+      fail-fast: false
+    env:
+      SYMFONY_DEPRECATIONS_HELPER: 'weak'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: pecl, composer
+          extensions: curl, openssl, mbstring
+          ini-values: memory_limit=-1
+      - name: Get composer cache directory
+        id: composercache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composercache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: ${{ runner.os }}-composer-
+      - name: Remove FOSUser
```

Comment:
```suggestion
      - name: Install FOSUser
```
?

Reply:
Oops yes, thanks! also I'm not sure if there is other way than `remove` and `require`.

Reply:
So it fails with the current range version & requires `dev-master`?

Reply:
let me try because I was trying too many things before and I'm not sure anymore 😬

Reply:
you're right, for some unknown reason I added that 🤦 thanks!